insertion_sort.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import unicode_literals
+from time import time
+
+
+def insertion_sort(list):
+    for index in range(1, len(list)):
+        value = list[index]
+        i = index - 1
+        while i >= 0:
+            if value < list[i]:
+                list[i + 1] = list[i]
+                list[i] = value
+                i = i - 1
+            else:
+                break
+
```

Comment:
Avoid using python reserved keywords as variables like the plague (list). This is a a doorway to bugs from hell.